src/Faker/Provider/Address.php

```diff
@@ -104,20 +104,24 @@ public static function country()
     }
 
     /**
-     * @example 77.147489
+     * @example '77.147489'
+     * @param float|int $min
+     * @param float|int $max
      * @return float Uses signed degrees format (returns a float number between -90 and 90)
      */
-    public static function latitude()
+    public static function latitude($min = -90, $max = 90)
     {
-        return static::randomFloat(6, 0, 180) - 90;
+        return static::randomFloat(6, $min, $max);
     }
 
     /**
-     * @example 86.211205
+     * @example '86.211205'
+     * @param float|int $min
+     * @param float|int $max
      * @return float Uses signed degrees format (returns a float number between -180 and 180)
      */
-    public static function longitude()
+    public static function longitude($min = -180, $max = 180)
```

Comment:
You're changing the results of existing calls. Please use 0 and 360 as min and max values.